.travis.yml

```diff
@@ -0,0 +1,20 @@
+language: java
+
+install:
+  - export PATH=/tmp/grpc-deps/bin:$PATH
```

Comment:
The build has not gotten to where LD_LIBRARY_PATH is necessary yet (due to failures). It is needed by our protoc plugin when it is exec'ed.